lib/modules/settingsConsole.js

```diff
@@ -1532,3 +1535,10 @@ function notifyOptionsSaved() {
 		setTimeout(() => $(statusEle).fadeOut(1000), 500);
 	}
 }
+
+function shortDescription(description) {
+	const tagIndex = description.indexOf('<');
```

Comment:
In order to avoid potential HTML parsing issues again, like https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/3574/files#r86195442, how about loading the description as an element's innerHTML, then storing that element's `textContent` as `firstText`?